mozdownload/scraper.py

```diff
@@ -816,6 +816,28 @@ def detect_platform(self):
 
         return platform
 
+    def is_build_dir(self, dir):
+        """Return whether or not the given dir contains a build."""
+
+        url = urljoin(self.base_url, self.build_list_regex, dir)
+
+        if self.application in MULTI_LOCALE_APPLICATIONS \
+                and self.locale != 'multi':
+            url = urljoin(url, self.locale)
```

Comment:
hmmm... trying to download a tinderbox build of b2g gave me the following error both in the PR (with commenting out of the section and without) and on the current master branch

>> /mozdownload$ ./scraper.py -a b2g -t tinderbox
Traceback (most recent call last):
  File "./scraper.py", line 1075, in <module>
    cli()
  File "./scraper.py", line 1070, in cli
    build.download()
  File "./scraper.py", line 281, in download
    if os.path.isfile(os.path.abspath(self.target)):
  File "./scraper.py", line 241, in target
    self.build_filename(self.binary))
  File "./scraper.py", line 199, in binary
    raise NotFoundError(message, self.path)
__main__.NotFoundError: Specified build has not been found: https://ftp.mozilla.org/pub/mozilla.org/b2g/tinderbox-builds/mozilla-central-l10n

So unless I have made a mistake in usage of the scraper for b2g (a MULTI_APPLICATION), this part of the code is broken and needs fixing... but that would be beyond the scope of this PR.

What is your take on it @whimboo. Taking the mentioned section out or leaving it in?